### PR TITLE
make the harness beautiful

### PR DESCRIPTION
**Pull request by johngruen**
This is updates styles to match the MVP and beautifies @lmorchard's harness page

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,11 +6,16 @@
   <link rel="stylesheet" href="/styles/main.css">
 </head>
 <body>
-  <div id="top-hits" class="result-region"></div>
-  <div id="search-suggestions" class="result-region"></div>
-  <div id="autocomplete-results" class="result-region"></div>
-  <div id="content-preview"></div>
-
+  <!--wrap all results or the :last-child slector targets script tags-->
```
Comment (nchapman) [inline src/index.html]: _slector_ typo

File: src/styles/main.scss [hunk 2]
```diff
@@ -9,29 +11,38 @@ html {
...
-  user-select: none
+  user-select: none;
 }
 
 body {
   font-family: message-box, sans-serif;
   margin: 0;
-  overflow: hidden;
   padding: 0;
+  background: $white;
```
Comment (nchapman) [inline src/styles/main.scss]: Out of order

**Comment by nchapman** [comment]
Looks great @johngruen. Feel free to merge after fixing those little things (I used my @pdehaan :eyes:).

**Comment by johngruen** [comment]
@nchapman, @pdehaan wouldn't have missed double line-breaks in sass :)

**Comment by nchapman** [comment]
> @pdehaan wouldn't have missed double line-breaks in sass

I absolutely knew that once I said that someone would point out that I missed glaringly obvious things that would make @pdehaan :crying_cat_face:.

File: src/styles/main.scss (hunk 2 above)
Comment (pdehaan) [inline src/styles/main.scss]: I love you guys.

**Comment by pdehaan** [comment]
@pdehaan would have asked why the CSS in [/src-harness/styles/**](https://github.com/mozilla/universal-search-content/blob/master/src-harness/styles/harness.css) uses mixed case (`#searchClient`) but the code in [/src/styles/**](https://github.com/mozilla/universal-search-content/blob/master/src/styles/main.scss) uses hyphens and lower case (`.result-region`).

**Comment by pdehaan** [comment]
(runs off to check [SCSS-Lint](https://github.com/brigade/scss-lint))

**Pull request merged**